src/main/java/duke/exception/ByEmptyException.java

```diff
@@ -0,0 +1,4 @@
+package duke.exception;
+
+public class ByEmptyException extends Exception {
```

Comment:
Instead of leaving the custom exception classes empty, you may want to implement custom exception messages in the new exception classes you created.

Reply:
Is it also recommended to throw all the exceptions to the main method to resolve them? I find myself catching the same type of exceptions twice (in two separate methods). If I catch all the exceptions in the main method, I only need to do this once per exception.